pennylane/templates/subroutines/out_adder.py

```diff
@@ -205,11 +206,6 @@ def map_wires(self, wire_map: dict):
             new_dict["work_wires"],
         )
 
-    @property
-    def wires(self):
-        """All wires involved in the operation."""
-        return self.hyperparameters["x_wires"] + self.hyperparameters["work_wires"]
-
```

Comment:
This should still return something. It just needs to check if there are work wires before returning.

Reply:
Safe to delete, after discussion